quiz-bundle-spec.md

```diff
@@ -0,0 +1,71 @@
+# Qwiklabs Quiz Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
+
+## `qwiklabs.yaml` Structure
+
+Here's a sample `qwiklabs.yaml` file with all nested details removed to make it easier to see the general file structure.
+
+```yml
+entity_type: Quiz
+schema_version: 1
+
+# The authoring title of the quiz - not visible to learners.
+title: Super Hard Quiz
```

Comment:
I think the fact that learners will never see this title is fairly significant and I'd prefer a more descriptive key like `internal_title` that clearly indicates that this is for internal bookkeeping purposes only.

Reply:
After thinking about my comment a bit more, I believe it will be fairly controversial. I would not strongly advocate to make the change and I realize that the change would cause the quiz spec to deviate from our other specs. However, I think it's an important point and would appreciate a bit of discussion around it.

Reply:
Hmm... that's actually a really good idea. I'm good with `internal_title` - @jgan or @joncass - any opinions?

Reply:
Yeah I like the idea, but `internal_title` doesn't feel descriptive enough... and I just spent 20 minutes on thesaurus.com trying to come up with something better to no avail.

Reply:
@jgan @supersam654 do we want to settle on `title` or `internal_title`? I'm unopinionated.

Reply:
I lean towards `title` but will defer if someone feels strongly

Reply:
In keeping with the game spec, how about authoring_title?

Reply:
We could also eschew `title` and just go with something like `short_description`

Reply:
I've replaced `title` with `id`, so this is now moot 😄